stdci.yaml

```diff
@@ -1,5 +1,6 @@
 substage:
   - openshift_3-9
```

Comment:
is there a point in keeping 3.9?

Reply:
Let's discuss it on today's meeting.
It can be removed in the following PR.